web/geo/generateWorld.ts

```diff
@@ -36,6 +36,10 @@ coordEach(fc, (coord) => {
   coord[1] = round(coord[1], 4);
 });
 
-validateGeometry(fc, config);
+// Only validate geometries if there are any changes to them
+const hasChanges = getTopojsonChanges(fc, config.OUT_PATH);
+if (hasChanges) {
+  validateGeometry(fc, config);
+}
```

Comment:
Turns out @VIKTORVAV99 also did this in https://github.com/electricitymaps/electricitymaps-contrib/pull/5197, so this change should be reverted here (see https://github.com/electricitymaps/electricitymaps-contrib/pull/5197#issuecomment-1474391690)